### ISSUE TITLE
Migrate Elevator Plugin

### ISSUE THREAD

**Issue by nlamprian** [ignitionrobotics/ign-gazebo#420]
Since the list of missing plugins is still long, I thought I would take on migrating the Elevator plugin. Are there any new requirements for it?

**Comment by chapulina** [comment]
Thanks!

One difference I can think of is that now we can use an `ignition::msgs::Int32` message instead of using a string message and converting to an int.

Also, you can probably leverage the `JointPositionController` system to do the PID control.

**Comment by nlamprian** [comment]
I started by creating a new, more feature-rich, model.

I configured all non-moving links as kinematic. It behaves as expected in Gazebo 9, but not in Ignition Dome.

![elevator-model](https://user-images.githubusercontent.com/2074154/97812891-d46b6180-1c84-11eb-913f-8ab269aa8882.gif)

The model can be found [here](https://app.ignitionrobotics.org/nlamprian/fuel/models/Elevator).

@chapulina is this a bug, or is the `kinematic` property not implemented yet?
